includes/Modules/Search_Console.php

```diff
@@ -371,14 +371,28 @@ protected function create_search_analytics_data_request( array $args = array() )
 		if ( ! empty( $args['end_date'] ) ) {
 			$request->setEndDate( $args['end_date'] );
 		}
+
+		// Only include data for URLs that are part of the current site.
+		$filter = new Google_Service_Webmasters_ApiDimensionFilter();
+		$filter->setDimension( 'page' );
+		$filter->setOperator( 'contains' );
+		$filter->setExpression( esc_url_raw( untrailingslashit( $this->context->get_reference_site_url() ) ) );
```

Comment:
The `untrailingslashit` is redundant here since this is always applied for this method, and this is covered by a test:
https://github.com/google/site-kit-wp/blob/1c9f13947cba8d9953ca57b9e87d05c1993d212c/tests/phpunit/integration/ContextTest.php#L151-L153

```suggestion
		$filter->setExpression( esc_url_raw( $this->context->get_reference_site_url() ) );
```